pom.xml

```diff
@@ -5,8 +5,8 @@
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>groupId</groupId>
-    <artifactId>Kinesis-sparkStreamingSource</artifactId>
-    <version>1.0.0</version>
+    <artifactId>kinesis-streaming-source</artifactId>
```

Comment:
this is a backwards incompatible change.

Reply:
though I agree that the current name isn't very good, just making sure the implications are clear.

Reply:
thought about that - how do we handle these? Should I bump the version to 2.0.0?

too bad we didn't catch this during code review.

Reply:
have reverted the artifact name change right now.